### ISSUE TITLE
Rework Sailing decisions to To&Tu Activities 

### ISSUE THREAD

**Issue by Remerod** [jj248/RealmsInExile#909]
With Tours and Tournament implementing Activities (In the free patch) it has been mostly agreed upon that our Sailing Activities would be better made into Activities. 

> Jamie:
> Sailing in definitely a good candidate for a travel activity. Plus we can change the sailing trait to be tiered lifestyle with XP like the others. It'd also solve the issue we have now of having coastal Rhûners sailing to Ray or whatever.

So This issue is for Rework our Sailing decisions into the new Activities that came with Tours and Tournament free patch

**Comment by Remerod** [comment]
I think the First thing that you should choose in the Activites is from which harbour you should travel into the sea from
Sidenote: we should rework harbour themed buildings to give us some extra stuff for the activites.

so that you can travel to that harbour, if your not already there. Then the real question is should you also pick the end goal, should that be decided at the harbour with an event (kind of how it is now?) or should that be predetermined when making the activity? and then maybe there should be some **points of interest** (introduced in ck3 patch 9.1) in the ocean maybe?  
(**points of interest** explained)
<details>
<summary>Click to expand</summary>

![image](https://github.com/jj248/RealmsInExile/assets/79544635/dc24f580-2507-4188-abdc-b3857677a218)
if you walk on a barony with a **points of interest** on it~

![image](https://github.com/jj248/RealmsInExile/assets/79544635/a5cae1d7-def7-4b66-9f29-a2ac208030f1)
you'll get a reward like 200 martial lifestyle points

</details>

If we do have **points of interest** in the ocean we could change this to have it level a mariner trait (that we now can gain experiences in with To&Tu). Also certain **points of interest** also can start events (according to Vector going onto Rome and Bagdad will do this in vanilla but i haven't tested that yet) So having certain points in the ocean could be very cool, and we could also have it that if your going somewhere you could rest at certain harbours (that could have **points of interest** or not)

I see a lot of potential here and differant ways of implementing them :)

**Comment by JukeJukestar** [comment]
Regarding Rome and Bagdad in vanilla: I have compatched it to be Bôzisha-Dâr and Korlea (since the loc ended up being partially fitting, though I rewrote most of it anyways)

Basically, if you choose to stop there on a travel you can get this special event once in your life. Couple different options for what to do, nothing too fancy.

**Comment by BriceLeduc** [comment]
This most likely won't be adapted for 3.1, moving to 4.0

**Comment by Jaco-Daan** [comment]
Realistically this can't be done at the moment. The key issue we have is that:

1. The whole 'Sailing to Meneltarma' chain will either need to be seperate, which feels very weird or we will need to add sea tile provinces all the way to Meneltarma.
2. Currently, there is no way to limit in which order the travel happens via the GUI window, i.e., they can select any province they want, and they can also edit their route, meaning they might not even go via the ocean.


TLDR; suggesting we close this until we have a more viable option either via GUI wizardry or via some jank.

Thoughts @jj248 @VectorMaximus @BriceLeduc

**Comment by jj248** [comment]
Yeh... my strong preference is to leave it as it is for now. It should still be functional! If we wanted to do anything we could make sailing a tiered trait instead, but that's about the limit of my ambition for it right now!

**Comment by Jaco-Daan** [comment]
Currently not really possible due to how travel locations working, therefore closing until this is actually possible
